tests/fixtures/basic-js/.sauce/config.yml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1alpha
+kind: folio
+sauce:
+  region: us-west-1
+  metadata:
+    name: Playwright Hello World
+    tags:
+      - e2e
+      - release team
+      - other tag
+    build: Release $CI_COMMIT_SHORT_SHA
+docker:
+  image:
+    name: saucelabs/stt-playwright-jest-node
+    tag: latest
+playwright:
+  version: "1.4.1"
```

Comment:
I think this ought to be specified on the suites level, since that's the only way to control what browser version is going to be used.

Reply:
@alexplischke this is meant to be the framework version.